openresty-build-tools/kong-ngx-build

```diff
@@ -474,6 +490,12 @@ main() {
           OPENRESTY_OPTS+=('--with-luajit-xcflags="-DLUAJIT_USE_VALGRIND -DLUA_USE_ASSERT -DLUA_USE_APICHECK -DLUAJIT_USE_SYSMALLOC"')
         fi
 
+	if [ $PROCESSOR_TYPE == "s390x" ]; then
+          # Change the configure file for s390x
+          echo "Modifying configure for s390x"
+          sed -i '742,785s/.*/#&/;717,726s/.*/#&/' ./configure
+        fi
+
```

Comment:
Need to add a version check here, as these changes only work for OpenResty 1.17.8.1 and 1.17.8.2.